pkg/osvscanner/osvscanner.go

```diff
@@ -671,6 +671,10 @@ func filterPackageVulns(r reporter.Reporter, pkgVulns models.PackageVulns, confi
 	var newVulns []models.Vulnerability
 	if len(newGroups) > 0 { // If there are no groups left then there would be no vulnerabilities.
 		for _, vuln := range pkgVulns.Vulnerabilities {
+			if isUnimportant(pkgVulns.Package.Ecosystem, vuln.Affected) {
```

Comment:
Can you aggregate the number of ignored entries and log that number (similar tto how we log ignored entries above).

Reply:
Also, I'm not sure how common this will be, but if there're other items in the vuln group (I.e. aliased or share aliases), should we be ignoring all of them?